tensorboard/plugins/scalar/vz_line_chart/vz-line-chart.ts

```diff
@@ -244,7 +289,12 @@ Polymer({
     if (this._chart) {
       this._chart.setTooltipPosition(this.tooltipPosition);
     }
-  }
+  },
+  _computeTooltipTableHeaderHtml(tooltipColumns) {
+    // The first column contains the circle with the color of the run.
+    const mapper = (c: ChartHelpers.TooltipColumn) => `<th>${c.title}</th>`;
```

Comment:
**Please sanitize all values here.**

For now, you can copy-and-paste the `safeRun` escaping code from the scalar dashboard. We can consider exposing it from a shared module later.